.travis.yml

```diff
@@ -0,0 +1,58 @@
+language: python
+python:
+  - "3.7"
+
+branches:
+  only:
+    - master
+    - nightly-tests
+
+git:
+  depth: 1
+
+before_install:
+  # Decrypt secrets
+  # Secrets generated using the CLI: `travis encrypt-file --add [...]`
+  - openssl aes-256-cbc -K $encrypted_36785d287aa1_key -iv $encrypted_36785d287aa1_iv
+    -in ci/keys/substra-208412-3be0df12d87a.json.enc -out ci/keys/substra-208412-3be0df12d87a.json
+    -d
+  # Install helm (v2.14.2)
+  - curl https://get.helm.sh/helm-v2.14.2-linux-amd64.tar.gz -o helm-v2.14.2-linux-amd64.tar.gz
+  - tar xzf helm-v2.14.2-linux-amd64.tar.gz
+  - mv linux-amd64/helm linux-amd64/tiller $HOME/bin/
+  # Install kubectl
+  - curl -LO https://storage.googleapis.com/kubernetes-release/release/`curl -s https://storage.googleapis.com/kubernetes-release/release/stable.txt`/bin/linux/amd64/kubectl
+  - chmod +x ./kubectl
+  - mv ./kubectl $HOME/bin/
+  # Install skaffold
+  - curl -Lo skaffold https://storage.googleapis.com/skaffold/releases/v1.7.0/skaffold-linux-amd64
+  - chmod +x ./skaffold
+  - mv ./skaffold $HOME/bin/
+
+script:
+  - CLUSTER_NAME="substra-tests-$(date -u +'%Y-%m-%d-%Hh%M')"
+  - cd ci/
+  # Retry on failure. This is to account for the occasional hiccups in deployment of hlf-k8s/substra-backend (some pods occasionally fail to start, maybe 1 time out of 10-20).
+  - travis_retry python -u ./run-ci.py --keys-directory=./keys/ --cluster-name=${CLUSTER_NAME}
+
+notifications:
+  slack:
+    rooms:
+      # TODO: send to the correct channel:
+      # - change target channel in the slack UI
+      # - regenerate encrypted payload below (https://docs.travis-ci.com/user/notifications/#configuring-slack-notifications)
+      - secure: UC3111xkSw2nluYtQb4Uqpufkc51WeY4pbm3TJdKJ5ydcTUrvDpcYHic8/5xfBLgU2EkBN6795bu9CNSZHDqO43QkMVSfz3bBea3S3r/bTDng33FaWUMaU1qLI2YnLrlO9STEL0bFGWc0OiC0BPW8kJvKYkk6szukB8OdvOt0mqApNcTRGkiofAlewk1ot1m1vTmMMryZF0cyTWyYz10dxacGSOIvILJECNP+JqpnjRzCat5sfjUjVS5jESBAIw++I3sLLSp7QTgDjY2SpIeEh0ksd6rVZUx2I1FAG3XbhlCAqWeqp6gG6CJbzl6q9Dm9cKgUWPlw5zBQ5mwl+Qqncui/Nt5BSa+F/WFxpYKKgoIIhBJvobpsZ96U0pIMpr5hZSPjs2axRPSSFjt8aiEp4fqojFj+50AXgp0qip0dZoG41n4VVCtaHHxwZUVhJ48Hkqv2gWH4pT56oYoZylT+E9fKAuFhDm4f0WeQdXo9pB1cqEM1xSHMIX8M/YiXxNDZE073MjReXVsArhqHQwoGBIP6ITYRPwUjqargnuUY6A/SoelV4e+mU2iyLjyaXNmx+LvQL+wnrP9yDXI2KsovFLZpetpm7arPramu/jAJE+d+wxqUPnq22TOOrtoZCGZbFn98D+fbjCzEw+b/slovdsMjCDcYAoYp7HttqA0fMM=
+    on_success: change
+    on_failure: always
+    template:
+      - "Nightly test <%{build_url}|#%{build_number}> of %{repository_name}@%{branch} %{result} in %{duration}"
+    if: type = cron AND (branch = master OR branch = nightly-tests) AND fork = false
+
+  email:
```

Comment:
could be pleasant to have a subject pattern here to be able to filter nightly-tests emails

Reply:
Looks like this isn't supported unfortunately: https://config.travis-ci.com/ref/notifications/email